accountClient/accountClientV2.go

```diff
@@ -0,0 +1,56 @@
+package accountClient
+
+import (
+	"context"
+
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/authClient"
+)
+
+//HTTP PATH prefix for calls to the e3db Account service for v1 and v2
+const (
+	AccountServiceV2BasePath = "v2/account"
+)
+
+//E3dbAccountClient implements an http client for communication with an e3db Account service.
+type E3dbAccountClientV2 struct {
+	APIKey    string
+	APISecret string
+	Host      string
+	*authClient.E3dbAuthClient
+}
+
+// ServiceCheckV2 checks whether the account service V2 is up and working.
+// returning error if unable to connect service
+func (c *E3dbAccountClientV2) ServiceCheckV2(ctx context.Context) error {
```

Comment:
no V2 suffix needed for these methods now that they're part of a separate client